Wit/WITCircleView.m

```diff
@@ -12,22 +12,18 @@ @interface WITCircleView ()
 @end
 
 @implementation WITCircleView
-#pragma mark - Layout
-- (void)layoutSublayersOfLayer:(CALayer *)layer {
-    if (layer == self.layer) {
-    }
-}
 
 #pragma mark - Lifecycle
-- (id)init {
+- (instancetype)init {
     self = [super init];
-    if (self) {
-        self.circleLayer = (WITCircleLayer*)self.layer;
+    if (!self) {
+        return nil;
+    }
```

Comment:
This is not a very common pattern in my experience. Can we stick to
```
self = [super init];
if (self) {
  // setup
}
```

Or, if you want to save a line:
```
if (self = [super init]) {
  // setup
}
```